### ISSUE TITLE
Improve design of site-example

### ISSUE THREAD

**Issue by ihorzenich** [AleshaOleg/holy-grail-markup#1]
Create a „redesign“ for our „Holy-grail example“ in some modern, 2017-year like look.
Implementation of this will be an case study for restyling site using different methodologies.

**Comment by AleshaOleg** [comment]
@ihorzenich then the old markup (which is ongoing), we will leave a separate branch?

**Comment by ihorzenich** [comment]
@AleshaOleg yes, and we should think about - how to demonstrate problems we're going to face working on it for each methodology.

**Comment by idmytro** [comment]
https://philipwalton.github.io/solved-by-flexbox/demos/holy-grail/

**Comment by AleshaOleg** [comment]
@cisar great example, would you like to implement new markup at least for one of methodology? Also, what about menu tabs? Is any design for them?

**Comment by idmytro** [comment]
default bootstrap 4 https://getbootstrap.com/docs/4.0/components/navs/#vertical is ok

**Comment by AleshaOleg** [comment]
yes, it should be ok.
